draft-ietf-rats-architecture.md

```diff
@@ -787,16 +787,25 @@ in order to bootstrap the sequence.
 The Verifier trusts (or more specifically, the Verifier's security
 policy is written in a way that configures the Verifier to trust) a
 manufacturer, or the manufacturer's hardware, so as to be able to
-appraise the trustworthiness of that manufacturer's devices.  In solutions
-with weaker security, a Verifier might be configured to implicitly
-trust firmware or even software (e.g., a hypervisor).  That is, it
-might appraise the trustworthiness of an application component, operating
+appraise the trustworthiness of that manufacturer's devices.
+
+In solutions with weaker security, a Verifier might be configured to implicitly
+trust firmware or even software (e.g., a hypervisor).
+That is, it might appraise the trustworthiness of an application component, operating
 system component, or service under the assumption that information
-provided about it by the lower-layer hypervisor or firmware is true.
+provided about it by the lower-layer firmware or software is true.
 A stronger level of assurance of security comes when information can be vouched
 for by hardware or by ROM code, especially if such hardware is
-physically resistant to hardware tampering.  The component that is
-implicitly trusted is often referred to as a Root of Trust.
+physically resistant to hardware tampering.
+The component that is implicitly trusted is often referred to as a Root of Trust.
```

Comment:
I think the above text applies only to local attestation because of the phrase "implicitly trust".  

Since the main subject of this document is remote attestation, I think this paragraph should be move down several paragraphs and clearly prefaced to indicate it is about local attestation that is distinct from the main topic of remote attestation.

Reply:
I think all of the text above is actually about remote (and local, but the main scope is remote) attestation.

Reply:
While the text refers to HW & SW, as well as elaborates on a specific security considerations unique to a "hardware root of trust", I think it is very much applicable to remote attestation as a whole.

I'd like to see feedback on a specific question, though:

How is a conveyance of data without any "cryptographic proof of trustworthiness" still an attestation procedure at all? "Local" attestation is typically realized via policies that reside in HW roots of trusts. These policies are deployed on an Attester by a trusted party and are semantically close related with the duties of a Verifier, I think (therefore the "local" attestation).

Reply:
When there is local trust between the Attester and Verifier it is because they are on the same chip, the same bus or such. The connection between them is trusted without crypto.

The use of the word "implicit" is what makes it local to me. In remote Attestation, the trust is explicit. The Attester manufacturer tells the Verifier it can trust the Attester explicitly.